streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -39,10 +51,12 @@ protected Windows() {
     }
 
     /**
-     * Set the window maintain duration in milliseconds of streams time.
+     * Set the window maintain duration (retention time) in milliseconds.
```

Comment:
We have public method `maintainMs()` that returns retention time and I would like to mirror this connection in the JavaDocs. Or we rename the method to `retentionMs()`, but this would need a KIP. For now, I would leave it as is, but will put the renaming ides into the uber API KIP I am going to do.

Does this make sense?